opentracing-mock/src/main/java/io/opentracing/mock/MockTracer.java

```diff
@@ -95,6 +93,28 @@ public synchronized void reset() {
         return new ArrayList<>(this.finishedSpans);
     }
 
+    /**
+     * @return a copy of all finish()ed Traces(Spans) started by this MockTracer grouped by traceId and spanId in HashMap format.
+     */
+    public Map<String, Map<String, Span>> finishedTraces() {
```

Comment:
It should return a MockSpan not span

Reply:
I thought It is better to use a more general return data type.